File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    // asks users for height
    do
    {
        height = get_int("Height: ");
    }
    // asks height if height < 1 and Height > 8
    while (height < 1 || height > 8);

    for (int i = 0; i < height; i++)
    {
        // adds space which starts by one less than height
        for (int j = i; j < height - 1; j++)
        {
            printf(" ");
        }
        // prints mario blocks according to height starts from 1
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        printf("  ");
        // prints mario blocks according to height starts from 1
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        // Adds new line to the code
        printf("\n");
    }
}

File: pset6/python_Exercise/mario.py
from cs50 import get_int
import sys

while True:
    height = get_int("Height: ")
    if (height >= 1 and height <= 8):
        break
# prints Mario Block
for i in range(height):
    # adds space
    for j in range(1, height - i):
        print(" ", end="")
    # adds # as per the block
    for k in range(i + 1):
        print("#", end="")

    print('\n', end="")
